### DISCUSSION TITLE
Domotizzare tapparelle su rete zigbee

### DISCUSSION THREAD

**Discussion by UgoCo** [indomus/forum#1800] [HUB: Home Assistant]
Salve, avrei l'esigenza di domotizzare due tapparelle, io uso homeassistant su Raspberry P4 e per evitare di sovracaricare la rete wifi, utilizzo la rete zigbee, ora vi chiedo gentilmente un consiglio su cosa comprare per domotizzare le due tapparelle sempre zigbee.
Grazie a chi mi risponderà

**Comment by UgoCo** [comment]
nessun consiglio?

**Reply by KC-inDomus** [reply]
il mio è di usare [Shelly 2.5](https://indomus.it/componenti/shelly-2-5/). non saturi niente e stai ben più sicuro.

**Comment by UgoCo** [comment]
si ok, ma volendo qualcosa che sfrutta la rete zigbee?

**Reply by snakuzzo** [reply]
non fare reply aprendo un nuovo thread...rispondi sotto l'ultimo messaggio.

Comunque dai un'occhiata [QUI](https://www.zigbee2mqtt.io/supported-devices/#s=shutter) per farti un'idea
